src/components/new-request/name-check/resources/name-check-base-items-conflicts.ts

```diff
@@ -36,11 +36,11 @@ export const baseItemsConflicts = {
   [NameCheckItemType.EXACT_MATCH]: {
     icon: 'mdi-alert-octagon',
     iconColor: 'red darken-2',
-    problem: 'This <b>exact name</b> is currently used by an existing BC entity or reserved for use',
+    problem: 'This <b>exact name</b> is currently in use or reserved for use by a BC corporation',
```

Comment:
Can you please check with UX team on whether to use "BC" or "B.C."? I see conflicting requirements between various tickets/uxpins.

Reply:
Meng, you can merge after the above is decided/fixed, or you can merge now and fix the above in a second PR later.

Reply:
Name request is using "BC", please see my comments in original ticket, I'll update after UX team decide what to use.